app/controllers/blog_controller.rb

```diff
@@ -1,4 +1,3 @@
 class BlogController < ApplicationController
-  def index
-  end
+  def index; end
```

Comment:
- [ ] You use SQLITE as database adapter but For this project it is required to use PostgreSQL as Database adapter Please fix it in order to make your app work as expected :+1: